src/ConfigHelper.ts

```diff
@@ -83,6 +83,7 @@ let NATIVE_THREAD_PROPS_WHITELIST: Record<string, boolean> = {
   fontSize: true,
   lineHeight: true,
   textShadowRadius: true,
+  textShadowOffset: true,
```

Comment:
I know its not related to you PR directly, but
1️⃣ Do you think we should have some checks to verify that there are no common properties in UI_THREAD_PROPS_WHITELIST and NATIVE_THREAD_PROPS_WHITELIST?
2️⃣ Do you think we should update type `Record<string, boolean>` and replace `string` with `keyof ViewStyle | TextStyle | ...` and `boolean` with `true`?

Reply:
🥇 That's a good idea, we can add a check in `configureProps` function.
🥈 While using `boolean: true` seems a good idea, the former proposition doesn't sound good. If we do a dependency on `ViewStyle | TextStyle | ...` we risk that types for this part of code will break depending on changes in React Native - some props might be gone, or added.